manifests/pipecd/templates/configmap.yaml

```diff
@@ -163,6 +167,11 @@ data:
           - source_labels: [__meta_kubernetes_pod_container_port_name]
             action: keep
             regex: admin
+        metric_relabel_configs:
+          - source_labels: [grpc_service]
+            target_label: pipecd_grpc_service
+            regex: pipe.api.service.(.+)service.(.+)
```

Comment:
For instance:
- pipe.api.service.apiservice.APIService
- pipe.api.service.webservice.WebService
- pipe.api.service.pipedservice.PipedService